lib/zclient.h

```diff
@@ -312,12 +318,28 @@ struct zapi_pw_status {
 	uint32_t status;
 };
 
+enum zapi_route_notify_owner {
+	ZAPI_ROUTE_FAIL_INSTALL,
+	ZAPI_ROUTE_BETTER_ADMIN_WON,
+	ZAPI_ROUTE_INSTALLED,
+};
+
 /* Zebra MAC types */
 #define ZEBRA_MAC_TYPE_STICKY                0x01 /* Sticky MAC*/
 #define ZEBRA_MAC_TYPE_GW                    0x02 /* gateway (SVI) mac*/
 
 /* Prototypes of zebra client service functions. */
 extern struct zclient *zclient_new(struct thread_master *);
+
+#if CONFDATE > 20181101
+CPP_NOTICE("zclient_new_notify can take over or zclient_new now");
```

Comment:
I think there is a spelling mistake here.

Reply:
fixed